### DISCUSSION TITLE
Make it weather proof

### DISCUSSION THREAD

**Discussion by djtrao** [adonno/tagreader#210] [Q&A]
Hi guys, I need some inspiration. 
I bought the Sammich and so far was able to set up exactly what I wanted: 
Home Assisant integration, then: => scan a nfc tag => open door via Nuki Pro 3.0

So my endgoal is to have nfc tags I can give my young children, in the case they come home from school and for whatever reason my wife and I are not home yet. It'll save me the headache of giving them keys and having to change locks if they lose them. Also it could be a system, where I would give a cleaning lady one (and it'd only work on days where she comes to clean) and so on. 

So: for this I need to position the tag reader on the outside of the house. The tripple glas windows are too thick, to read the tag from the other window side. So the reader really has to be outside. It's no biggie, just will have the usb cable go outside, and power it. 

But: I need a way to water/weather proof it. I'm sure, if I just leave it outside - even when not exposed to direct rain, the humidity will break it sooner or later. 

What ideas can you guys come up, for an easy DIY weather proofing? First I thought of a little plastic box (but would have to find one that size, would suck to have one that's too big). Next I thought... a balloon? Just pull the balloon over the reader, have the cable come out and even there, really close it tightly. No humidity should enter? 

Sorry for the long text, but I thought maybe it might inspire someone to use their tag for a simliar thing.

**Comment by adonno** [comment]
The case won't resist UV for too long since it's basic PLA. 
there is electrical gel you could use but that could remain "liquid", maybe dip the PCB in vaseline ?  isn't conducting either . The ballon idea isn't that bad even though not the nicest. And finally there are special resins for outside use, you coul dopen the case and pour resin in it but, the i won't be able to provide further support. 

these are ideas not suggestions. :-D

**Reply by djtrao** [reply]
I like that alot (but am not handy enough I think for some of these haha).
The cool thing is: It just has to be inside of "Something". Could be something black/whatever... so UV protecting too. Since it's not a must, that we can see it, the NFC's reach is about 7cm I'd say. So that probably would work through 2-3cm of plastic, easy. 

Remember the old times, when we had actual film rolls for our photo cameras? These round, black pastic thingies? Something liket that in a square would be great. Plastic cover to close it, and just have the cable come out, close that whole with simple glue gun (I'm handy enough for that one lol)

**Reply by adonno** [reply]
sourcing map 83 x 58 x 33 mm electronically waterproof IP65 sealed ABS plastic DIY junction box housing clear https://amzn.eu/d/4m6Fny0

Something like that would really make it safe

**Reply by djtrao** [reply]
Genius!!! that's it. Probably even going to get one that is not transparent. That'll make it even more secure, less obvious.

**Comment by m-kloeckner** [comment]
Depending on where you live you should not worry about humidity too much. Just make sure that the case is waterproof.

I built a DIY rain gauge with a sensor that is meant to be used indoors and it works just fine for >2 years now.

**Comment by wout1976** [comment]
Is there a way to make te signal stronger? It does not go to my isolated windows. And outdoor is not an good idea
